sdk/cmake/baseaddress.cmake

```diff
@@ -181,9 +181,9 @@ set(baseaddress_itss                           0x745e0000)
 set(baseaddress_joy                            0x745a0000)
 set(baseaddress_jscript                        0x74400000)
 set(baseaddress_jsproxy                        0x743d0000)
-set(baseaddress_kbdeo                          0x743b0000)
-set(baseaddress_kbdrost                        0x74390000)
-set(baseaddress_kbdsf                          0x74370000)
+#set(baseaddress_kbdeo                          0x743b0000) #unwanted, should vanish when rerunning the script
+#set(baseaddress_kbdrost                        0x74390000) #unwanted, should vanish when rerunning the script
+#set(baseaddress_kbdsf                          0x74370000) #unwanted, should vanish when rerunning the script
```

Comment:
Then, why not just manually deleting those lines? (in this baseaddress.cmake file and the others)

Reply:
I could have done this also yes. But by commenting them, we do still see the "gaps" in the base-addresses-ranges. Next time the script will be run, they will be gone.

Reply:
The lines are left for verifying the effectiveness of the changes when running the script later.